hack/dependency-stats.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@dims is this really worth checking in? this is a new potential source of merge conflicts, we can just run the tool as needed?

if we need to track it over time we can upload the results somewhere in a postsubmit CI job.
Unless we plan to restrict the values here, I don't think this should be part of the update/verify model / checked-in.

Reply:
cc @liggitt